### ISSUE TITLE
Drawer Submenu Items

### ISSUE THREAD

**Issue by ryanrox** [leocb/MaterialSkin#177]
I was just wondering if anyone had thought about making the drawer have an option for specific items to have a dropdown submenu that could be toggled to view. It would be handy for navigating through nested tabs like Account: Overview, Details. 
I'm not sure how to implement this or if it would be too much of a pain, but I thought I'd just throw that out there.

**Comment by orapps44** [comment]
Do you mean something like this :

![image](https://user-images.githubusercontent.com/77468294/120906296-82dee700-c658-11eb-86a5-b1f2be7803b0.png)

**Comment by ryanrox** [comment]
Yes something just like that would be awesome! Maybe also a small drop-down arrow on the right side of Item one that indicates whether or not that item is selected. I'm not sure if there would be an issue with too many items based on drawer/form size or if the need for a scrollbar/mousewheel capability for Item or Sub items like KristanSmout's pull request suggests would be necessary.

EX:
![1](https://user-images.githubusercontent.com/84647797/121028159-30dbb580-c76d-11eb-9969-8b3915764d53.png)
![2](https://user-images.githubusercontent.com/84647797/121028161-30dbb580-c76d-11eb-850a-b651adc0f797.png)
![3](https://user-images.githubusercontent.com/84647797/121028162-30dbb580-c76d-11eb-928c-deeb38fb2501.png)

**Comment by orapps44** [comment]
Right, drawer scroll is needed first (#162 ).
To be honest I don't plan to work on that topic, so if you (or anyone else) want to contribute please do.
